scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Copyright (C) 2017-2021 HERE Europe B.V.
```

Comment:
Commit message: typo in "reading multiple scan **resulst**"

Reply:
Done.